src/NoTimeToWaitBundle/Resources/views/Default/nttw4.html.twig

```diff
@@ -80,8 +80,7 @@ Closing: 17:00</p>
 
 <h2>Travel Grants</h2>
 
-<p>As done at past No Time to Wait conferences, we are offering travel grants to potential attendees that may need support. Thanks in particular to <a href="https://reto.ch/">AV Preservation by reto.ch</a>, for sponsoring these travel grants. Applications for travel grants shall be accepted until September 25th with announcements to recipients by September 30th. Please apply using <a href="http://bit.ly/nttw4travelgrant">our travel grant application form</a>.</p>
-<p>UPDATE: The deadline for travel grant applications has been extended to September 29th.</p>
+<p>As done at past No Time to Wait conferences, we are offering travel grants to potential attendees that may need support. Thanks in particular to <a href="https://reto.ch/">AV Preservation by reto.ch</a>, for sponsoring these travel grants. And congratulations to Brianna Toth ,Helia Pereira Marcal, Maral Mohsenin, and Michael Norman as our grant recipients.</p>
```

Comment:
Space at the wrong place: in`Toth ,Helia`.

Reply:
fixed, thx